### ISSUE TITLE
[Outreachy Task Submission] Accessibility Issue with the 'Forgotten Password?' button in Login Screen

### ISSUE THREAD

**Issue by Chiemezuo** [ushahidi/platform#4766]
# Intro
This is the second accessibility issue I found with the 'Forgotten Password?' button in the login screen. For the first, see #4757 

## Issue
Low contrast for the `Forgot password?` button and the login & signup when not in focus. ***This is a violation of WCAG Success Criterion 1.4.3 (Level AA) - Contrast (Minimum)***. [See](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum)

## Description
***Success Criterion 1.4.3 (Level AA) - Contrast (Minimum)*** requires that the visual presentation of text and images of text must have a contrast ratio of at least 4.5:1, except for large text, images of text, and incidental text.

## Impact
This means that some visually-impaired people will have trouble seeing the text at all, while some will have difficulty identifying it as a call-to-action.

## Fix
Improve the contrast ratio to at least 4.5:1. This will require a design decision by the team, and involves testing with an automated tool.
